app/services/param_to_shql.rb

```diff
@@ -30,16 +30,18 @@ def call
 
       facet_filters = []
       @filter_param.each do |label,facet|
+        facet_filter = []
         if facet.keys.include?("gt") || facet.keys.include?("lt")
           range_filter =  range_param_to_shql(label, facet)
-          facet_filters << range_filter if !(range_filter == "" || range_filter == nil)
+          facet_filter << range_filter if !(range_filter == "" || range_filter == nil)
```

Comment:
Could we change `if !(range_filter == "" || range_filter == nil)` to `range_filter.present?' for better readability?